src/lib/utils.ts

```diff
@@ -582,3 +581,9 @@ export function formatDecodedOffer(
     quantityMax
   }
 }
+
+export function convertVersionNumber(version: string): number {
+  const [withoutDash] = version.split('-')
+  const withoutDots = withoutDash.replace('.', '')
+  return Number(withoutDots)
+}
```

Comment:
```suggestion
export function convertVersionNumber(version: string): number {
  const [withoutDash] = version.split('-')
  const withoutV = withoutDash.replace(/^v/, '') // Trim "v" from start of the string
  const withoutDots = withoutV.replace('.', '')
  return Number(withoutDots)
}
```

My version comes in as `v22.11.1`

Reply:
Ah yeah these versions are tricky as they seem to come back in so many different formats!